### ISSUE TITLE
How will data flow into this site?

### ISSUE THREAD

**Issue by jlooper** [fempire/fempire-vue#1]
Do we need a flat file, or some sort of Firebase integration? If we create a form that people can fill out we can more easily sort alphabetically, and reduce data entry and format errors

**Comment by ladyleet** [comment]
@jlooper yeah i literally have no idea. i mean... do we have one json file that lives someplace that is updated as the MD file is updated or something? we probably need to make this more amazing. i still love the idea of people contributing via PR so I want to keep that... but I'm not totally committed to this idea.
